### DISCUSSION TITLE
Error ENOENT

### DISCUSSION THREAD

**Discussion by OowarriorwhiteoO** [heliomarpm/udemy-downloader-gui#126] [General]
15/5/2023 15:38:09 - errorDownload: Error: ENOENT: no such file or directory, unlink 'C:\Ultimate Python de cero a programador experto\1. Introducción\2. REPL de Python.mp4'
15/5/2023 15:38:08 - errorDownload: Invalid file path:  C:\/Ultimate Python de cero a programador experto/1. Introducción/2. REPL de Python.mp4.mtd

HOW CAN I SOLVE THIS ERROR?

**Comment by heliomarpm** [comment]
> 15/5/2023 15:38:09 - errorDownload: Error: ENOENT: no such file or directory, unlink 'C:\Ultimate Python de cero a programador experto\1. Introducción\2. REPL de Python.mp4'
> 15/5/2023 15:38:08 - errorDownload: Invalid file path:  C:\/Ultimate Python de cero a programador experto/1. Introducción/2. REPL de Python.mp4.mtd
> 
> HOW CAN I SOLVE THIS ERROR?
> 

Close the application, delete the .mtd file and try again
